cluster_hosts/tasks/get_cluster_hosts_target.yml

```diff
@@ -79,12 +79,11 @@
         - name: get_cluster_hosts_target/aws | Assert that snapshots exists
           assert:
             that:
-              - item in _available_snapshots
+              - _available_snapshots|length == cluster_hosts_target|length

```

Comment:
So we're now only checking whether the number of snapshots matches the number of hosts.  Are you content to remove the check on the requested snapshots being available?

Reply:
As I understand this change will fit both cases: snapshots need to be available AND match the number of hosts. Is there another edge case this wouldn't fulfil?

Reply:
I don't think so... the original code checks that the snapshot in cluster_vars (e.g. `123456`) is also found in the returned `ec2_snapshot_info`.  The new code just checks there are the same number of snapshots in each.  @mikekrasn - can you confirm either way?

Reply:
Original code in it's nutshell is basically checking that [ec2_snapshot_info](https://github.com/sky-uk/clusterverse/blob/master/cluster_hosts/tasks/get_cluster_hosts_target.yml#L69) is working correctly with filtering tags. New code 'trusts' that ec2_snapshot_info has successfully fetched all the snapshots by provided filters and compares that infrastructure is of the same size as the number of fetched snapshots(in case that someone provides too broad tags)

Reply:
That was the original question: are you happy that you're removing one check, and replacing it with another.  If so, fine, (but at least change the `name:` because `Assert that snapshots exists` is not what it is doing anymore.

Reply:
I have changed the assertion description to clarify what's been tested.